### DISCUSSION TITLE
Multiple TQDM progress bars: You probably need to run some code to see this widget

### DISCUSSION THREAD

**Discussion by miohtama** [sagemathinc/cocalc#6709] [Q&A]
I have a notebook that is running several TQDM based progress bars in Jupyter notebook mode (`from tqdm.auto import tqdm`).

For some reason I am getting a warning "You probably need to run some code to see this widget." for some of the progress bars. 

<img width="1757" alt="image" src="https://github.com/sagemathinc/cocalc/assets/49922/3bdb19aa-9c96-4980-a467-31a8962574c6">

I am also getting different errors like

<img width="1765" alt="image" src="https://github.com/sagemathinc/cocalc/assets/49922/502ef15b-1cd0-449d-abc1-765facc2abe3">

It seems to work flakily, though, so I am not sure what's the root cause here.

I have a public notebook available if this needs to be issued tracker and repeated.

**Comment by williamstein** [comment]
This is https://github.com/sagemathinc/cocalc/issues/6128 unfortunately.

**Comment by miohtama** [comment]
I read the ticket. Based on the discussion on the ticket, is there is a safe older ipywidgets version which one can downgrade to?

I double checked `ipywidgets` version:

```shell
docker-compose exec cocalc pip3 list
```

```test
ipywidgets                      7.7.5
```

**Reply by williamstein** [reply]
Something before 7.x.   

This ticket is high priority though, so hopefully I'll have fixed this soon.  I may get to tackle it this coming week.
